tests/Serializer/SerializerTest.php

```diff
@@ -95,11 +96,45 @@ public function testNull(bool $serializeAllObjects): void
         $this->assertNull($result);
     }
 
+    public function testLongStringWithOverwrittenMessageLength(): void
+    {
+        $serializer = $this->createSerializer(new Options(['truncation_length' => 500]));
+
+        foreach ([100, 490, 499, 500, 501, 1000, 10000] as $length) {
+            $input = str_repeat('x', $length);
+            $result = $this->invokeSerialization($serializer, $input);
+
+            $this->assertInternalType('string', $result);
+            $this->assertLessThanOrEqual(500, \strlen($result));
+        }
+    }
+
+    public function testClippingUTF8Characters(): void
+    {
+        if (!\extension_loaded('mbstring')) {
```

Comment:
Since we're requiring the `mbstring` extension in the `composer.json` we can safely remove this code